CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing to the Lacework Go-sdk
+
+#### Table Of Contents
+
+[Before getting started?](#before-getting-started)
+
+[How to Contribute?](#how-can-i-contribute)
+* [Reporting Bugs](#reporting-bugs)
+* [Feature Requests](#feature-requests)
+* [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+* [Git Commit Messages](#commit-message-standard)
+
+
+## Before getting started
+
+Read the [README.md](https://github.com/lacework/go-sdk/wiki/CLI-Documentation), the [development documentation](https://github.com/lacework/go-sdk/tree/main/cli#development) 
+and the Lacework CLI documentation on the [wiki](https://github.com/lacework/go-sdk/wiki/CLI-Documentation).
+
+## Reporting Bugs
+
+Ensure the issue you are raising has not already been created under [issues](https://github.com/lacework/go-sdk/issues).
+
+If no current issue addresses the problem, open a new [issue](https://github.com/lacework/go-sdk/issues/new).
+Include as much relevant information as possible. See the [bug template](https://github.com/lacework/go-sdk/) for help on creating a new issue.
+
+## Feature Requests
+
+If you wish to submit a request to add new functionality or an improvement to the go-sdk then use the the [feature request]() template to 
+open a new [issue](https://github.com/lacework/go-sdk/issues/new)
+
+## Creating a Pull Request
+
+When submitting a pull request follow the [commit message standard](#commit-message-standard).
```

Comment:
Feel this needs fleshed out more, instead of jumping straight to commit messages